example.js

```diff
@@ -24,9 +30,12 @@ async function main() {
     }
   });
 
+
   // Initialize the robot connection
   await sdk.connectRobot({ robotId, name: 'robot0' });
 
+  sdk.registerCommandCallback(robotId, logCommand);
+
```

Comment:
As a lazy-programmer, I want to do `sdk.registerCommandCallback(logCommand);` to listen for commands in any already or to-be connected robot, so I don't need to register the callback for each of them